docs/integration/slack.md

```diff
@@ -33,6 +33,7 @@ in the right channels.
     **Repositories** on the right, then click the **Link** button.
 -   You can always add, edit and remove channel links in the **Manage
     > Integrations > Slack** page.
+-   `@atomist` will automatically be invited to linked channels.
```

Comment:
Are we not calling it "the Atomist bot" or "the `@atomist bot" anymore?

Reply:
I debated that. Slack labels it as `APP` in channel messages since it's technically not a bot, however, I think that "The `@atomist` bot" is probably clearest and most widely understood.